cypress/integration/smoke-test.js

```diff
@@ -1,8 +1,13 @@
 beforeEach(function () {
     cy.fixture('postcodes.json').as('postcodes')
-    Cypress.Cookies.defaults({
-        whitelist: ['_get_help_to_retrain_session', 'seen_cookie_message']
-    });
+    // Cypress.Cookies.defaults({
+    //     whitelist: ['_get_help_to_retrain_session', 'seen_cookie_message']
+    // });
+})
```

Comment:
@mahmoodc I think the last change needed here is to revert this as discussed for now.